aten/src/ATen/native/native_functions.yaml

```diff
@@ -1956,17 +1957,33 @@
     SparseCPU: mul_out_sparse_cpu
     SparseCUDA: mul_out_sparse_cuda
     MkldnnCPU: mkldnn_mul_out
+    QuantizedCPU: quantized_mul_out
   supports_named_tensor: True
 
   # For C++ only, until we have conversion from C++ numbers to Tensor
 - func: mul.Scalar(Tensor self, Scalar other) -> Tensor
   use_c10_dispatcher: full
   variants: function, method
+  dispatch:
+    CPU: mul
+    CUDA: mul
+    QuantizedCPU: quantized_mul
 
 - func: mul_.Scalar(Tensor(a!) self, Scalar other) -> Tensor(a!)
   use_c10_dispatcher: unboxed_only
   variants: method
 
+- func: mul_relu.Tensor(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
```

Comment:
can you not make it a function? and maybe make the namespace for the function to be 'python_module: nn' (then it'd be torch._C._nn.mul_relu? I don't think we should expose `torch.mul_relu` as top level function and definitely not `Tensor.mul_relu`

Reply:
We do not need to rush this for 1.3, scalar add and mul do occur in a few models, but I havent yet seen a mul_relu or an add_relu for scalars yet.

To be cleaner, should we also have a float variant of this function so that the API does not diverge between float and quantized? 
Also agree that we should not expose fused operations in torch. namespace